helm/distributed-idol/templates/content/statefulset.yaml

```diff
@@ -0,0 +1,113 @@
+# BEGIN COPYRIGHT NOTICE
+# (c) Copyright 2022 Micro Focus or one of its affiliates.
+# END COPYRIGHT NOTICE
+{{ $component := .Values.content }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ $component.name }}
+  labels: {{- include "distributedidol.labels" . | nindent 4 }}
+spec:
+  serviceName: {{ $component.name | quote }}
+{{- if .Values.setupMirrored }}
+  replicas: {{ int .Values.content.initialEngineCount }}
+{{- else }}
+  replicas: {{ int .Values.content.initialEngineCount }}
+{{- end }}
```

Comment:
I see this was taken more or less verbatim from the original content-set.yaml, but are these alternatives supposed to be identical (and if so, what's the purpose of the if/else)?